### DISCUSSION TITLE
Allow plugin to be added/removed without requiring a restart

### DISCUSSION THREAD

**Discussion by jjw24** [Flow-Launcher/Flow.Launcher#507] [Ideas]
Allow plugin to be installed/uninstalled without needing to reload all via restarting the entire flow app.

![image](https://user-images.githubusercontent.com/26427004/102327640-21f42180-3fda-11eb-9dcc-8b5eeba5be38.png)

[DotNetCorePlugins](https://github.com/natemcmaster/DotNetCorePlugins), outstanding [issue](https://github.com/natemcmaster/DotNetCorePlugins/issues/141) preventing usage

**Comment by taooceros** [comment]
Seems that .net core has supported dynamic assembly unload
https://docs.microsoft.com/en-us/dotnet/standard/assembly/unloadability
